internal/cmd/options.go

```diff
@@ -43,6 +45,15 @@ func (opts *LoggingFlagOpts) ApplyFlags(fs *pflag.FlagSet) {
 
 // SetLoggerLevel sets up the logger based on common options.
 func (opts *LoggingFlagOpts) SetLoggerLevel(ctx context.Context) context.Context {
+	logger := log.GetLogger(ctx)
+	ctx = executableTrace.WithExecutableTrace(ctx, &executableTrace.ExecutableTrace{
+		ExecuteStart: func(executableName, action string) {
+			logger.Info("started executing credential helper program %s with action %s", executableName, action)
+		},
+		ExecuteDone: func(executableName, action string, err error) {
```

Comment:
If the execution done with error, should we log in `error` level.

Reply:
Added `if err == nil` check and used `error` level accordingly.